docs/dcp2_system_design.rst

```diff
@@ -1121,6 +1121,20 @@ Once a data release is published, the snapshots that make up the release may
 never be deleted. Superseded snapshots that were removed from the release prior
 to publishing should be deleted.
 
+All snapshots that make up a release must reside in the same `BigQuery
+location`_ so that they can be indexed simultaneously by Azul using a single
+BigQuery reservation. Reservations are location-specific. The location of a
+particular snapshot may change as long as the previous condition is satisfied
+for all releases.
+
+|nn| This would mean that snapshots can be relocated, but that all snapshots in
```

Comment:
what would be the use case for relocating snapshots?  It's not a feature that exists -- so trying to understand the driving forces

Reply:
This is not saying that snapshots have to be relocated, just that they can. I don't think it is far fetched to consider the case. If snapshots never have to be relocated, great. If they do, this points out a not so obvious consequence of the normative section right above.

Reply:
One use case would be that a particular region offers better cost or CO2 footprint so we might want to move snapshots to that region. Keep in mind that query jobs run in the location of the tables.

Reply:
short answer: as long as we are saying they "can" and not that they should, I'm all good!  

Long answer: Every use case I've seen around snapshot location is data policy driven -- it needs to live in a specific countries boundary or something along those lines. I think cost is also an important driver, but that most likely means we use cold storage like mechanisms to save cost, but agreed that cost will be a use case that impacts how objects are stored in the cloud